content/sections/section2.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "Blog"
+weight: 3
+improvecontrast: true
+---
+
+## AMU-OSS Blog
+
+You can read our blogs here.
+
+{{< big-button text="BLOG" href="https://testamuossblog.tk/read" >}}
```

Comment:
This blog link won't work.

Reply:
what should be the link here?

Reply:
@RJ722 should we add the blog link? I don't think the blog website you created is still working.